flathunter/chrome_wrapper.py

```diff
@@ -59,13 +59,12 @@ def get_chrome_driver(driver_arguments):
     """Configure Chrome WebDriver"""
     logger.info('Initializing Chrome WebDriver for crawler...')
     chrome_options = uc.ChromeOptions() # pylint: disable=no-member
-    if platform == "darwin":
```

Comment:
Did you deliberately remove this? What happens if you add it back? (I'm testing on Linux, so I don't use this code path).